packages/truffle-core/lib/command.js

```diff
@@ -100,6 +100,35 @@ class Command {
       }
     });
 
+    //Check unsupported command line flag according to the option list in help
+    try {
+      const inputOptions = [];
+      inputStrings.map(i => {
```

Comment:
This is a really nitpicky style thing but how would you feel about being a little more verbose with this variable name...maybe name it `inputString`? I think it reads a little bit better even though it probably doesn't much matter.